smoke_zephyr/configuration.py

```diff
@@ -202,6 +202,63 @@ def save(self):
 		with open(self.configuration_file, 'w') as file_h:
 			file_h.write(self._serializer('dumps', self._storage))
 
+class MemoryConfiguration(Configuration):
+	"""
+	This class provides a generic object for parsing information from variables in multiple formats.
+	"""
+
+	def __init__(self, mem_object, object_type, prefix=''):
+		"""
+		:param str mem_object: The memory object to parse.
```

Comment:
`mem_object` should be a python object that responds to `__getitem__` not a string. If it's a string, the caller should de-serialize it appropriately before passing it here.